tests/unit/lms/services/lti_grading/factory_test.py

```diff
@@ -0,0 +1,56 @@
+from unittest.mock import Mock, sentinel
+
+import pytest
+
+from lms.services.lti_grading.factory import factory
+
+
+class TestFactory:
+    def test_v11(
```

Comment:
Again, cheating here a bit. I'm testing

`lms.services.lti_grading.factory` but asserting the behavior of `lms.services.lti_grading._v11|v13.factory`.

I personally think mocking `_v11.factory` here and having an independent test for it on the `_v11` file wound't add much value to the test. I'm using the fact that the other modules start with an underscore as a justification.